Update VRC-14 with data access focus until TDVL threshold reached (#36)
vana-com/VRCs: VRCs/vrc-14.md

   - TDVL (Total Data Value Locked) is the sum of all assets locked in DataFi applications on Vana, including DEX liquidity pools, lending protocols, and other applications built on Vana. Total Data Value Locked is similar to Total Value Locked (TVL) on other chains. 
2) There is no longer a top 16, so any DLP which meets the eligibility requirement qualifies for incentives. For example, there could be 30 DLPs eligible for incentives. To be eligible, DLPs must meet all criteria of VRC-5, have migrated to the new data access architecture, and have a token compliant with the VRC-20 standard. 
3) For this example, assume the DLP incentives pool is 300,000 VANA for the epoch, and 30 DLPs are eligible. The split of rewards is determined at the end of the epoch based on the following three performance metrics: DLP token trading volume, unique data contributors, and data access, measured as averages across the epoch. 

>    **Until $1B TDVL is reached, only the data access fees portion (50%) of the rewards pool is distributed. The remaining 50% allocated to DLP token trading volume and unique data contributors is carried forward to future epochs, and not assigned to any DLP.**

Author: ```suggestion
   **Until $1B TDVL is reached, only the data access fees portion (50%) of the rewards pool is distributed. The remaining 50% allocated to DLP token trading volume and unique data contributors is carried forward to future epochs, and not assigned to any DLP. Once $1B TDVL is reached, these carried forward rewards will be included in the full balanced scoring system distribution.**
```

### Removal of Top 16
Eliminating the "top 16" limitation and extending epochs to quarterly cycles promotes collaboration over competition. DLPs can focus on their unique value proposition rather than competing for limited slots in the top 16 ranking system. Quarterly epochs allow DLPs to implement more sophisticated strategies and reduce short-term optimization that might sacrifice long-term sustainability. The removal of the top 16 also allows for specialized DLPs serving various domains from robotics to healthcare to thrive in parallel. 

### Focus on Data Access Until $1B TDVL

> Until the ecosystem reaches $1B in Total Data Value Locked, only 50% of the rewards pool (the data access fees portion) is distributed, with rewards based solely on data access performance. This approach ensures that during the critical early growth phase, DLPs are incentivized to focus on building datasets that generate actual value through consumption. Once the ecosystem reaches sufficient scale at $1B TDVL, the full balanced scoring system activates and the complete rewards pool is distributed.
> 

Author: ```suggestion
Until the ecosystem reaches $1B in Total Data Value Locked, only 50% of the rewards pool (the data access fees portion) is distributed, with rewards based solely on data access performance. This approach ensures that during the critical early growth phase, DLPs are incentivized to focus on building datasets that generate actual value through consumption. Once the ecosystem reaches sufficient scale at $1B TDVL, the full balanced scoring system activates and the complete rewards pool is distributed.

The $1B threshold represents a maturity milestone where the ecosystem has demonstrated sufficient scale and liquidity to support a more complex reward structure. Before this point, concentrating rewards on data access fees creates the strongest incentive alignment for building valuable, consumable datasets rather than optimizing for metrics that may not translate to real utility.
```

### Removal of Top 16
Eliminating the "top 16" limitation and extending epochs to quarterly cycles promotes collaboration over competition. DLPs can focus on their unique value proposition rather than competing for limited slots in the top 16 ranking system. Quarterly epochs allow DLPs to implement more sophisticated strategies and reduce short-term optimization that might sacrifice long-term sustainability. The removal of the top 16 also allows for specialized DLPs serving various domains from robotics to healthcare to thrive in parallel. 

### Focus on Data Access Until $1B TDVL

> Until the ecosystem reaches $1B in Total Data Value Locked, only 50% of the rewards pool (the data access fees portion) is distributed, with rewards based solely on data access performance. This approach ensures that during the critical early growth phase, DLPs are incentivized to focus on building datasets that generate actual value through consumption. Once the ecosystem reaches sufficient scale at $1B TDVL, the full balanced scoring system activates and the complete rewards pool is distributed.

Author: ```suggestion
Until the ecosystem reaches $1B in Total Data Value Locked, only 50% of the rewards pool (the data access fees portion) is distributed, with rewards based solely on data access performance. This approach ensures that during the critical early growth phase, DLPs are incentivized to focus on building datasets that generate actual value through consumption. Once the ecosystem reaches sufficient scale at $1B TDVL, the full balanced scoring system activates as trading volume and contributor metrics become more meaningful indicators of a mature, diversified DataFi ecosystem.
```
